### PR TITLE
Add classification datasets (eng, ron, swe)

### PR DESCRIPTION
**Pull request by mmhamdy**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

- This PR adds 3 Classification datasets for English, Romanian, and Swedish

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

File: mteb/tasks/Classification/swe/SwedishSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["swe-Latn"],
+        main_score="accuracy",
+        date=("2021-01-01", "2022-01-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
```
Comment (x-tabdeveloping) [inline mteb/tasks/Classification/swe/SwedishSentimentClassification.py]: Are we sure it doesn't have a license? If not specified can you please write `"not specified"` ?

File: mteb/tasks/Classification/swe/SwedishSentimentClassification.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 1024
```
Comment (x-tabdeveloping) [inline mteb/tasks/Classification/swe/SwedishSentimentClassification.py:6]: Is this necessary? I would prefer if you just wrote 1024 twice

File: mteb/tasks/Classification/eng/PoemSentimentClassification.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        dataset={
+            "path": "poem_sentiment",
+            "revision": "329d529d875a00c47ec71954a1a96ae167584770",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2020-01-01", "2021-01-01"),
```
Comment (x-tabdeveloping) [inline mteb/tasks/Classification/eng/PoemSentimentClassification.py:21]: This doesn't seem right? Are we sure that all of the poems are from this one year period? If not, can you come with a reasonable estimate for which years the poems can be from?

File: mteb/tasks/Classification/eng/PoemSentimentClassification.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="CC-BY-4.0",
+        socioeconomic_status="mixed",
```
Comment (x-tabdeveloping) [inline mteb/tasks/Classification/eng/PoemSentimentClassification.py:26]: I don't know much about the corpus at hand, but at least in some countries in Europe I know that poetry was very much a thing of the elites/nobility. Are we sure that `mixed` is the appropriate label here?

**Review by x-tabdeveloping — CHANGES_REQUESTED** [review]
Looks okay, I would look into the dates a bit more thoroughly though

File: mteb/tasks/Classification/swe/SwedishSentimentClassification.py (hunk 1 above)
Comment (mmhamdy) [reply mteb/tasks/Classification/swe/SwedishSentimentClassification.py]: I couldn't find any. The HF Repo says "Research only" which is not a real license and there was none on github, so I kept it as "Not specified".

File: mteb/tasks/Classification/swe/SwedishSentimentClassification.py (hunk 2 above)
Comment (mmhamdy) [reply mteb/tasks/Classification/swe/SwedishSentimentClassification.py:6]: I just was asked to do it this way on previous PRs.

File: mteb/tasks/Classification/eng/PoemSentimentClassification.py (hunk 3 above)
Comment (mmhamdy) [reply mteb/tasks/Classification/eng/PoemSentimentClassification.py:21]: Those are randomly picked verses from the Project Gutenberg. They could span a very wide time period (possibly starting from 1700s), so I approximated the date of collection. What do you suggest here?

File: mteb/tasks/Classification/eng/PoemSentimentClassification.py (hunk 4 above)
Comment (mmhamdy) [reply mteb/tasks/Classification/eng/PoemSentimentClassification.py:26]: Yeah, I agree, and that is what I thought at the beginning. But this is a diverse collection from different countries that could belong to various demographies.

File: mteb/tasks/Classification/swe/SwedishSentimentClassification.py (hunk 2 above)
Comment (x-tabdeveloping) [reply mteb/tasks/Classification/swe/SwedishSentimentClassification.py:6]: alright then

File: mteb/tasks/Classification/eng/PoemSentimentClassification.py (hunk 3 above)
Comment (x-tabdeveloping) [reply mteb/tasks/Classification/eng/PoemSentimentClassification.py:21]: I would give a rough estimate of the date range of creation, not collection, I think that's kind of the point of the attribute

Comment (mmhamdy) [reply mteb/tasks/Classification/eng/PoemSentimentClassification.py:21]: I've updated it. But this is a rough estimate.

**Review by x-tabdeveloping — APPROVED** [review]
Please correct the points file, otherwise it's all good.

**Pull request merged**